fakeaudit/utils.go

```diff
@@ -14,51 +15,97 @@ import (
 
 var absPath, _ = filepath.Abs("../")
 
-func ReadConfigFiles(filename string) runtime.Object {
+func ReadConfigFiles(filename string) (decoded []runtime.Object, err error) {
 	buf, err := ioutil.ReadFile(filename)
 
 	if err != nil {
 		log.Error("File not found")
 		panic(err)
```

Comment:
Since there is an output err parameter we don't need to `panic` here just return would be good enough as it would get handled subsequently. 

Or we handle all the errors in this function and remove the output err parameter